### DISCUSSION TITLE
[Native Image]Java Swing Application

### DISCUSSION THREAD

**Discussion by ryan1-chang** [oracle/graal#3599] [Q&A] [answered]
I could get native-image to build an exe from the HelloWorld Swing app.
When I execute the exe, can I hide that window?

![image](https://user-images.githubusercontent.com/61516352/126034275-c4e194a0-ecc6-4755-bf61-80f0a7fa8b7f.png)

**Comment by chirontt** [comment, marked as answer]
By default, the MainApp.exe is created as a console application in Windows; this is why you see the command window popping up in the background when launching MainApp.exe. To eliminate it, you should convert your MainApp.exe to a GUI application, using the editbin utility from MS VS installation:

`editbin /subsystem:windows MainApp.exe`

**Reply by ryan1-chang** [reply]
**This solution is ok, thank you very much**
